ktor-server/ktor-server-plugins/ktor-server-metrics-micrometer/jvm/src/io/ktor/server/metrics/micrometer/MicrometerMetrics.kt

```diff
@@ -194,7 +222,9 @@ public val MicrometerMetrics: ApplicationPlugin<MicrometerMetricsConfig> =
 
         application.monitor.subscribe(RoutingRoot.RoutingCallStarted) { call ->
             call.attributes.getOrNull(measureKey)?.let { measure ->
-                measure.route = call.route.parent.toString()
+                measure.route = call.route.let { route ->
+                    pluginConfig.transformRoute(route)
+                }
```

Comment:
Minor. Let's drop `.let { ... }` here
```suggestion
                measure.route = pluginConfig.transformRoute(call.route)
```